README.md

```diff
@@ -1 +1,9 @@
-# hello-world
\ No newline at end of file
+# hello-world
+
+Top 5 Rappers of 2020
+
+1. JCole
+2. Travis Scott
+3. Drake
+4. Big Sean
+5. Dom Kennedy
```

Comment:
I think Dom Kennedy should be replaced as well :/ He's got some classics, but not in 2020.

Reply:
How do you feel about Meg? She's got some hits and her tiny desk performance was one of a kind.

Reply:
I love that idea! Go ahead and replace Dom Kennedy with Meg.